packages/plans-grid-next/src/hooks/data-store/types.ts

```diff
@@ -0,0 +1,23 @@
+import { FeatureList, PlanSlug, TERMS_LIST } from '@automattic/calypso-products';
+import { AddOnMeta } from '@automattic/data-stores';
+import { UseCheckPlanAvailabilityForPurchase } from '@automattic/data-stores/src/plans';
```

Comment:
Can we also please import directly from `@automattic/data-stores` and not the `src` folders? We mentioned this earlier, can we confirm if we are adopting this everywhere?

Also, for consistency's sake, we can: `import { AddOns, Plans } from '@automattic/data-stores';` and grab their types/hooks from there. Or do you prefer not to? 🤔

Reply:
I generally defer to my IDE for imports, but it seems I need to get in the habit of checking them for Calypso. I've https://github.com/Automattic/wp-calypso/pull/87478/commits/0b691b336cbdaef6e88f0262ce850b8cfe2d48d2 this instance and didn't find any remaining `/src/` references.

Aside: the approach of using `Plans.xx` and `Sites.xx` rather than directly importing hooks/types etc is a bit unfamiliar to me. What's the motivation there?

Reply:
data-stores package is quite messy right now. I guess my attempt to bring some order and go from there. lots of things should be refactored out hopefully at some point